extras-requirements.txt

```diff
@@ -11,14 +11,13 @@
 boto3~=1.9, <1.17.107
 botocore>=1.20.106,<1.20.107
 aiobotocore~=1.4.0
-# aioitertools 0.9 and above requires typing-extensions>4, which conflicts with kfp
-aioitertools<0.9
+aioitertools
```

Comment:
You can just delete it. This line was only added because of the kfp issue.

Reply:
don't we want to make sure it's installed still?

Reply:
It's a transitive dependency, no need.